docs/changelogs/v0.36.md

```diff
@@ -29,6 +30,14 @@ go-log v2 has been out for quite a while now and it is time to deprecate v1.
 - Fixes `ipfs log tail`
 - Removes support for `ContextWithLoggable` as this is not needed for tracing-like functionality
 
+#### Bitswap Broadcast Reduction
+
+The bitswap client now supports logic to reduce the amount of bitswap broadcast messages that are send to peers. This logic is enabled by default.
+
+The bitswap broadcast reduction logic can be enabled and disabled and configured in the `Internal.Bitswap` section of the ipfs config file. For a description of the configuration items, see the `Internal.Bitswap.Broadcast...` variables in the documentation on [`Internal.Bitswap`](https://github.com/ipfs/kubo/blob/master/docs/config.md#internalbitswap).
+
+The overall logic works by sending to non-local peers only if those peers have previously replied that they have wanted data blocks. By default, broadcasts are always sent to peers on the local network.
+
```

Comment:
@gammazero is this an useful pointer for people who want to poke at it? 

```suggestion

To revert to the previous behavior for A/B testing, set `Internal.Bitswap.BroadcastNoReduction` to `false` and monitor relevant metrics (`ipfs_bitswap_bcast_skips_total`, `ipfs_bitswap_haves_received`, `ipfs_bitswap_unique_blocks_received`, `ipfs_bitswap_wanthaves_broadcast`). Depending on your workload, the number of broadcasts should decrease, while the block receipt success rate should remain acceptable.

```

Reply:
To revert to the previosu behavior and disable broadcast reduction, set `Internal.Bitswap.BroadcastNoReduction` to  `true`.

Maybe I should rename this option to `EnableBroadcastReduction` or `DisableBroadcastReduction` for clarity. Which do you prefer?

Reply:
Renamed to `BroadcastReductionEnabled` and made it a flag with default `true`

Reply:
Done